pkg/istio-agent/xds_proxy.go

```diff
@@ -571,8 +573,7 @@ func (p *XdsProxy) initDownstreamServer() error {
 	if err != nil {
 		return err
 	}
-	// TODO: Expose keepalive options to agent cmd line flags.
-	grpcs := grpc.NewServer(istiogrpc.ServerOptions(istiokeepalive.DefaultOption())...)
+	grpcs := grpc.NewServer(istiogrpc.ServerOptions(p.keepaliveOptions)...)
```

Comment:
Why do you need keepalive settings for the local uds connection? I don't see how that could break

Reply:
I understand uds does need keepalive, not sure how could block it